_config.yml

```diff
@@ -192,6 +192,12 @@ authors:
     github: yuki24
     twitter: yuki24
 
+  jonallured:
+    name: Jon Allured
+    site: http://jonallured.com
```

Comment:
HTTP? Not HTTPS? Tsk tsk 😛

Reply:
Guess I have some work to do!

https://www.digitalocean.com/community/tutorials/how-to-secure-apache-with-let-s-encrypt-on-ubuntu-16-04